### PR TITLE
ELA summative assessments shouldn't display trait scores unless trait…

### PR DESCRIPTION
**Pull request by malaffoon**
… reports are enabled; add docs about remote debugging containers

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -279,6 +279,45 @@ tenantProperties:
       name: Test Tenant
 ```
 
+#### Remote Debugging (from IDE)
```
Comment (malaffoon) [inline CONTRIBUTING.md:282]: I spent a while getting this working so i thought i'd capture the steps.

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts [hunk 2]
```diff
@@ -248,16 +248,17 @@ export class AssessmentResultsComponent implements OnInit {
...
-      (this._assessmentExam.assessment.type === 'sum' &&
-        this._assessmentExam.assessment.traitReportEnabled)
-    );
+    if (this._assessmentExam.assessment.type === 'sum') {
+      return this._assessmentExam.assessment.traitReportEnabled;
+    } else if (this._assessmentExam.assessment.subject === 'ELA') {
+      return this._assessmentExam.assessment.hasWerItem;
+    } else {
+      return false;
+    }
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.ts:261]: The way the logic was before, a Summative ELA assessment would show trait scores even if trait reports were disabled for that assessment. The new logic is more deliberate: *all* summatives use the trait report flag, *then* the hard-coded ELA check.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -279,6 +279,45 @@ tenantProperties:
...
+```
+2. Use `kubectl` to port forward the jvm debug port for the pod of interest. This needs a dedicated terminal
+(or you need to background the process). 
+```bash
+kubectl get po | grep myapp
+myapp-deployment-6d9f7fb7b-qmd5k     1/1     Running   0          17h
+
+kubectl port-forward myapp-deployment-6d9f7fb7b-qmd5k 5007:5007
+``` 
+3. Configure your IDE to remote debug to localhost on the specified port, `5007` in this example.
```
Comment (gregcharles) [inline CONTRIBUTING.md:321]: Interesting, so we could attach to a QA pod and debug against that with these instructions? Yes, that could be very useful! 

- Maybe there should be a disclaimer about issues with debugging against shared servers?
- "... there is an extra step," to me implies to take the above instructions and do this one more thing. That could be just me though. I'd probably put it as, "If you want to remote debug a kubernetes instance, the configuration is just slightly more complicated."

Comment (malaffoon) [reply CONTRIBUTING.md:321]: Good suggestions, i'll add those comments.

**Pull request merged**